docs/docs-content/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -0,0 +1,366 @@
+
+- [Validator](#validator) - Use the Validator open-source framework to validate the permissions required by Palette.
+
+## Static Placement
+
+Static placement is when you have a pre-existing network resource group, VNet, subnets and security groups you want Palette to use when deploying a cluster. Static placement supports three use cases. The required actions are all the same but the scope levels for each use case are different.
+
+
+<details>
+<summary>Expand to view all required actions</summary>
+
+Below are all the required actions for all static placement use cases without a specified scope level.
+
+- Microsoft.Compute/disks/delete
+- Microsoft.Compute/disks/read
+- Microsoft.Compute/disks/write
+- Microsoft.Compute/galleries/images/read
+- Microsoft.Compute/galleries/images/versions/read
+- Microsoft.Compute/virtualMachines/delete
+- Microsoft.Compute/virtualMachines/extensions/delete
+- Microsoft.Compute/virtualMachines/extensions/read
+- Microsoft.Compute/virtualMachines/extensions/write
+- Microsoft.Compute/virtualMachines/read
+- Microsoft.Compute/virtualMachines/write
+- Microsoft.Network/loadBalancers/backendAddressPools/join/action
+- Microsoft.Network/loadBalancers/delete
+- Microsoft.Network/loadBalancers/inboundNatRules/delete
+- Microsoft.Network/loadBalancers/inboundNatRules/join/action
+- Microsoft.Network/loadBalancers/inboundNatRules/read
+- Microsoft.Network/loadBalancers/inboundNatRules/write
+- Microsoft.Network/loadBalancers/read
+- Microsoft.Network/loadBalancers/write
+- Microsoft.Network/networkInterfaces/delete
+- Microsoft.Network/networkInterfaces/join/action
+- Microsoft.Network/networkInterfaces/read
+- Microsoft.Network/networkInterfaces/write
+- Microsoft.Network/networkSecurityGroups/read
+- Microsoft.Network/networkSecurityGroups/securityRules/delete
+- Microsoft.Network/networkSecurityGroups/securityRules/read
+- Microsoft.Network/networkSecurityGroups/securityRules/write
+- Microsoft.Network/privateDnsZones/A/delete
+- Microsoft.Network/privateDnsZones/A/read
+- Microsoft.Network/privateDnsZones/A/write
+- Microsoft.Network/privateDnsZones/delete
+- Microsoft.Network/privateDnsZones/read
+- Microsoft.Network/privateDnsZones/virtualNetworkLinks/delete
+- Microsoft.Network/privateDnsZones/virtualNetworkLinks/read
+- Microsoft.Network/privateDnsZones/virtualNetworkLinks/write
+- Microsoft.Network/privateDnsZones/write
+- Microsoft.Network/publicIPAddresses/delete
+- Microsoft.Network/publicIPAddresses/join/action
+- Microsoft.Network/publicIPAddresses/read
+- Microsoft.Network/publicIPAddresses/write
+- Microsoft.Network/routeTables/delete
+- Microsoft.Network/routeTables/read
+- Microsoft.Network/routeTables/write
+- Microsoft.Network/virtualNetworks/join/action
+- Microsoft.Network/virtualNetworks/read
+- Microsoft.Network/virtualNetworks/subnets/join/action
+- Microsoft.Network/virtualNetworks/subnets/read
+- Microsoft.Resources/subscriptions/resourceGroups/read
+
+</details>
```

Comment:
I'm not sure we're helping readers by also providing this entire list without the Palette-specific requirements. What was your intention behind it?